snapcraft/cli/_channel_map.py

```diff
@@ -0,0 +1,152 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2020 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from collections import OrderedDict
+from typing import List, Union, TYPE_CHECKING
+
+from tabulate import tabulate
+
+if TYPE_CHECKING:
+    from snapcraft.storeapi.v2 import snap_channel_map
+
+
+def _get_channel_hint(*, channel_map, fallback: str, architecture: str) -> str:
+    tick = "^"
```

Comment:
Any chance you can use a constant/enum for these tick types?

Reply:
yes I can!